packages/cozy-pouch-link/src/jsonapi.spec.js

```diff
@@ -31,28 +31,30 @@ const DELETED_DOC_FIXTURE = {
 const token = 'fake_token'
 const uri = 'https://claude.mycozy.cloud'
 const client = new CozyClient({ token, uri })
+client.capabilities = { flat_subdomains: true }
 
 describe('doc normalization', () => {
-  it('keeps the highest between rev and _rev and removes the rev attribute', () => {
```

Comment:
Is that you who removed this test? Or was I? I'm wondering if we should keep it.